src/main/java/edu/ithaca/dturnbull/bank/Account/CheckingAccount.java

```diff
@@ -0,0 +1,56 @@
+package edu.ithaca.dturnbull.bank.Account;
+
+import java.util.ArrayList;
+
+public class CheckingAccount extends AbstractAccount {
+    /**
+     * @throws IllegalArgumentException if email is invalid
+     */
+    public CheckingAccount(double startingBalance) {
```

Comment:
Constructor should call the AbstractAccount super() constructor. The same is true for many of your methods like deposit, transfer and withdraw since there is common functionality here.